app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/teidata/TEIDataFragment.kt

```diff
@@ -654,7 +654,7 @@ class TEIDataFragment :
             val intent = Intent(activity, EventInitialActivity::class.java)
             val bundle = Bundle()
 
-            bundle.putString(Constants.PROGRAM_UID, model.currentProgram().uid())
+            bundle.putString(Constants.PROGRAM_UID, model.currentProgram()?.uid())
```

Comment:
what could happen if this uid is null?. Could it provoke any error?. If so could we implement a way to inform the user at some point?